src/constants/networkMetadata.ts

```diff
@@ -14,20 +14,25 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+// This file is supposed to have default built-in metadata and should 
+// be upgraded in a timely manner regardless from metadata importing
+// tools.
+// TODO: automate updates
```

Comment:
I think it's better to add an issue than a TODO. If you want the todo in the code it should be "TODO: automate updates (issue: … …)"